.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 .awcache
 .cache
+.cache-loader
```

Comment:
shouldn't be needed w/ latest `yarn install` and all that (and once you do that you can `rm -rf .cache-loader` locally, it's all in `.cache` now)